documentation/versioned_docs/version-1.6.0/License and Bill of Materials

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
We should also add this file to the base documentation in [this folder](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/tree/main/documentation/docs). This is the folder which gets copied over to [versioned_docs](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/tree/main/documentation/versioned_docs) in case of a release.

Reply:
@didiladi so should I add the folder only to this location?